.travis.yml

```diff
@@ -163,11 +164,11 @@ jobs:
           - cd ..
 
     # Deploy the documentation on github (only for master branch).
-      deploy:        
+      deploy:
           provider: pages
           skip_cleanup: true
           local_dir: doc/html
           keep-history: true
           github_token: ${GH_REPO_TOKEN}
-          on: 
-          branch: master 
+          on:
+              branch: master
```

Comment:
Seems like this was indented incorrectly which caused PRs to deploy the docs :)

Reply:
I think we have a new maintainer 💃

Reply:
True thanks for finding :)